openapi/pom.xml

```diff
@@ -58,6 +58,10 @@
             <groupId>org.jboss</groupId>
             <artifactId>jandex</artifactId>
         </dependency>
+        <dependency>
+            <groupId>javax.xml.bind</groupId>
+            <artifactId>jaxb-api</artifactId>
+        </dependency>
```

Comment:
This might cause some complication with the newer JDKs (>= 11) where jaxb is not available.
You may need to use profiles to set jaxb-api with scope provided (< 11) or add the both the api and impl (>= 11)